CHANGELOG.md

```diff
@@ -6,6 +6,46 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
 
 ___Note:__ Yet to be released changes appear here._
 
+## 5.3.0
+
+### General
+
+* `FEAT`: add sticky group headers in properties panels ([#726](https://github.com/bpmn-io/bpmn-js-properties-panel/issues/726))
+* `FEAT`: allow application reload via keyboard shortcut ([#3089](https://github.com/camunda/camunda-modeler/issues/3089))
+* `FIX`: restore focus after modal and overlay close ([#2942](https://github.com/camunda/camunda-modeler/issues/2942))
+* `FIX`: properly highlight selected text in XML editor ([#2923](https://github.com/camunda/camunda-modeler/issues/2923))
+* `FIX`: remove user path from Mixpanel tracking ([#3105](https://github.com/camunda/camunda-modeler/pull/3105))
+* `DEPS`: bump @camunda/linting to v0.5.0 ([#3118](https://github.com/camunda/camunda-modeler/pull/3118))
+* `DEPS`: bump codemirror to v6.0.1 ([#3103](https://github.com/camunda/camunda-modeler/pull/3103))
+* `DEPS`: bump diagram-js to v8.9.0 ([#3113](https://github.com/camunda/camunda-modeler/pull/3113))
+* `DEPS`: bump @bpmn-io/properties-panel to v0.20.1 ([#3120](https://github.com/camunda/camunda-modeler/pull/3120))
+
+### BPMN
+
+* `FEAT`: show diagram errors on canvas ([#3118](https://github.com/camunda/camunda-modeler/pull/3118))
+* `FEAT`: add support for extension properties in Camunda Platform 8.1 ([#731](https://github.com/bpmn-io/bpmn-js-properties-panel/issues/731))
```

Comment:
We could add the lint rule as well: https://github.com/camunda/bpmnlint-plugin-camunda-compat/pull/43

Reply:
Yes, thanks, added!

> * `FEAT`: allow `zeebe:Properties` only in Camunda Platform 8.1 or higher ([#43](https://github.com/camunda/bpmnlint-plugin-camunda-compat/pull/43))